### PR TITLE
Feat/add nft section

### PR DESCRIPTION
**Pull request by hanakoori01**
### GH Issue

#430

### Steps to test
1. Correr `yarn build`
2. Correr `yarn serve`
3. Ir a `localhost:3000`
4. Ir a Pautas para Desarrolladores > Aprende EOSIO > NFTs en el ecosistema de EOSIO

#### CheckList
- [ ] Follow proper Markdown format
- [ ] The content is adequate
- [ ] The content is available in both english and spanish
- [ ] I Ran a spell check

### CODE REVIEW

**Review by xavier506 — APPROVED** [review]
LGTM

File: docs/aprender-eosio/nft-en-eosio.mdx [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+import { Grid, Box } from '@material-ui/core';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+Los activos digitales han incursionado en la vida cotidiana de cientos de personas. Utilidades como la verificación de productos, la trazabilidad o la autenticación son algunos de los hechos más importantes para los que se utilizan los tokens no fungibles.
+
+Para recorrer el tema puedes aprender los siguientes aspectos:
+
+## ¿Qué son los NFTs?
+
+NFT es la versión abreviada de Token No Fungible , una aplicación de cadena de bloques que representa la propiedad de cualquier activo físico o digital, tiene la característica de ser única y poco común. Música, videos y arte (digitales) Bienes raíces, joyería o automovilística son solo algunos ejemplos de piezas que pueden tokenizarse como Token No Fungible.
```
Comment (leisterfrancisco) [inline docs/aprender-eosio/nft-en-eosio.mdx:16]: is `tokenizarse` a word? consider `tokenizar`

File: docs/aprender-eosio/nft-en-eosio.mdx [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+			<td>
+				<a href="https://areaxnft.com/">
+					<img src="https://areaxnft.com/img/areax%20logo%20light.68b6831b.svg"/>
+				</a>
+			</td>
+			<td>
+                <p>AreaX NFT es un Marketplace basado en el estándar de la cadena de bloques de Telos.</p>
+            </td>
+			<td>
+                <p><strong>Tipo NFT:</strong> Arte digital, memes, musica, videos and más.</p>
```
Comment (leisterfrancisco) [inline docs/aprender-eosio/nft-en-eosio.mdx:61]: change `musica` to `música`

File: docs/aprender-eosio/nft-en-eosio.mdx [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+    <td>
+        <p>Proton, WAX y Telos tienen diferencias distinguibles sobre otras Blockchains. Una de las principales preocupaciones es la tarifa de gas por transacción, que es muy baja o nula. El ecosistema tiene esa ventaja.</p>
+    </td>
+    <td>
+        <p>Estos Market Places no reciben conexiones con las billeteras más populares como Metamask, CoinbaseWallet, Wallet Connect u otras.</p>
+    </td>
+    </tr>
+    <tr>
+    <td>
+        <p>Crear el NFT es un proceso en el que debe tener cuidado y estar preparado para administrar el proceso. La experiencia de subir tu criptoarte es una de las más fáciles, con no más de 5 pasos.</p>
```
Comment (leisterfrancisco) [inline docs/aprender-eosio/nft-en-eosio.mdx:110]: `criptoarte` or `cripto arte`?

**Pull request merged**